### ISSUE TITLE
FloatingPointError when running np.abs(np.nan)    (?!)

### ISSUE THREAD

**Issue by isofer** [pymc-devs/pymc3#127]
this is a weird one.
after I import pymc I get FloatingPointError whenever I run np.abs(np.nan).
(which apperatnly happens a lot when you work with pandas Dataframe)

```
>>python
Python 2.7.2 (v2.7.2:8527427914a2, Jun 11 2011, 15:22:34) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> np.abs(np.nan)
__main__:1: RuntimeWarning: invalid value encountered in absolute
nan
>>> np.abs(np.nan)
nan
>>> np.abs(np.nan)
nan
>>> import pymc as pm
>>> np.abs(np.nan)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FloatingPointError: invalid value encountered in absolute
```

I've looked for the RuntimeWarnnig on the web, and it seems not to be a problem. I ran np.abs(np.nan) a few times to show that the RuntimeWarning is not suppose to be the issue

I'm using Mac OS 10.6
numpy 1.6.2
pymc 2.2 (I've just try to reinstalled it using pip)

**Comment by twiecki** [comment]
I can't replicate:

``` python
In [1]: import numpy as np

In [2]: np.abs(np.nan)
Out[2]: nan

In [3]: import pymc as pm

In [4]: np.abs(np.nan)
Out[4]: nan
```

**Comment by fonnesbeck** [comment]
I also cannot replicate this.

**Comment by isofer** [comment]
found it. it's a numpy issue.
line 7 in `pymc/tests/__init__.py` was causing the problem.

```
numpy.seterr(divide = 'raise', invalid = 'raise')
```

```
In [1]: np.abs(np.nan)
/Library/Frameworks/Python.framework/Versions/2.7/bin/ipython:1: RuntimeWarning: invalid value encountered in absolute
  #!/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
Out[1]: nan

In [2]: numpy.seterr(divide = 'raise', invalid = 'raise')
Out[2]: {'divide': 'warn', 'invalid': 'warn', 'over': 'warn', 'under': 'ignore'}

In [3]: np.abs(np.nan)
---------------------------------------------------------------------------
FloatingPointError                        Traceback (most recent call last)
/Users/imrisofer/<ipython-input-3-512ec3c2657f> in <module>()
----> 1 np.abs(np.nan)

FloatingPointError: invalid value encountered in absolute
```

I'll check with the numpy guys

**Comment by isofer** [comment]
I found that this bug is related to the RuntimeWarning I got.
this is fixed in numpy 1.7.0
http://projects.scipy.org/numpy/ticket/1905
